### DISCUSSION TITLE
Feature for configuring "previous - next"

### DISCUSSION THREAD

**Discussion by gallrjbr** [GrandOrgue/grandorgue#1666] [General]
Hello, I'd like to ask if there is possibility to configure Grandorgue for recognizing toe studs or bottoms "previous - next" (change of memories)

**Comment by ClarkeBrunt** [comment]
Yes - definitely, so long as your toe stud sends a midi event.

Open the combination setter panel (from the 'Panel' menu), then right click on 'Next' or 'Previous' (and follow the displayed instructions) to program them to respond to your toe studs. Similarly all the other buttons in the panel.

There are also previous/next (and the current program number) displayed in the toolbar, but unless I'm missing something, you can't program them from there. The previous/next in the toolbar are completely equivalent to those in the combination setter panel.

If you have more than one control (say a thumb piston as well as a toe stud) which you want to do the same thing, then you can add more midi events to the same action.

See the 'help' about the toolbar and the combination setter panel if needed.
